admin/views/sign-in.ejs

```diff
@@ -10,7 +10,7 @@
 
         <p class="lede" id="lead-paragraph">Sign-in to access the Check Development Service</p>
 
-        <form action="/sign-in" method="post" aria-label="Sign-in below" autocomplete="off" autocapitalize="none">
+        <form action="/sign-in" method="post" aria-label="Sign-in below" autocomplete="off">
```

Comment:
Please either leave autocapitalise here or move it to the input fields.  It's needed on iOS tablets.

Reply:
@jon-shipley 'autocapitalize' comes up as invalid for element <form> - it throws a validation error (not warning). I found out that it's a "nonstandard attribute used by iOS Safari Mobile" - If you need it I'll put it back with a comment in both the markup and the README file.